dev-node/11ty+eleventy/11ty+eleventy-0.11.1.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
is this a good name for a package? could it be 11ty-eleventy?

Reply:
Original name is `@11ty/eleventy`

Reply:
> Original name is `@11ty/eleventy`

In that case I would just go with `eleventy`

Reply:
`eleventy` ia another package
https://www.npmjs.com/package/@11ty/eleventy
https://www.npmjs.com/package/eleventy